include/swift/AST/DiagnosticsParse.def

```diff
@@ -1860,5 +1860,9 @@ ERROR(expected_closure_literal,none,
 ERROR(expected_multiple_closures_block_rbrace,none,
       "expected '}' at the end of a trailing closures block", ())
 
+// Package declaration errors
+ERROR(package_declaration_not_allowed,none,
```

Comment:
This should probably provide a little more detail about why it is not allowed.

Reply:
🤔Since this is an experimental feature, we need some flag to allow such syntax, and this error will be thrown if these flags are not specified. I just didn’t come up with a better name?